src/adapters/trinidadtobago.js

```diff
@@ -0,0 +1,207 @@
+   */
+  const parseToJSON = (data) => {
+    try {
+      data['values'] = JSON.parse(data.values);
+      if (Object.keys(data['values']).length === 0) {
+        data = undefined;
+      }
+    } catch (e) {
+      data = undefined;
+    }
+    return data;
+  };
+  // Loops through all items
+  results.forEach(item => {
+
+    item = parseToJSON(item);
+    // If values are empty or something fails, dont run
+    if (item !== undefined) {
+      const template = {
+        city: item.meta.city,
+        location: item.meta.location,
+        parameter: Object.keys(item.values)[0],
+        coordinates: {
+          latitude: Number(item.meta.latitude),
+          longitude: Number(item.meta.longitude)
+        },
+        attribution: [{name: 'EMA', url: 'https://ei.weblakes.com/RTTPublic/DshBrdAQI'}],
+        averagingPeriod: {unit: 'hours', value: 1}
+      };
+      // Units are mostly ug/m3, but CO is mg/m3, according to site
+      template['unit'] = (template['parameter'] === 'CO') ? 'mg/m3' : 'ug/m3';
+      // Loops through the latest data for 24 hours, data is hourly
+      for (let i in item.values[template.parameter]) {
+        // Do not add values if values are Null
+        if (item.values[template.parameter][i] !== null) {
+          let m = Object.assign({value: item.values[template['parameter']][i]}, template);
+          // Adds the formated date
+          const dateMoment = moment.tz(item.values.xlabels[i], 'YYYY-MM-DD HH', 'America/Port_of_spain');
+          m['date'] = {
+            utc: dateMoment.toDate(),
+            local: dateMoment.format()
+          };
+          // unifies parameters and measurement units
+          m = unifyParameters(m);
+          m = unifyMeasurementUnits(m);
+          measurements.push(m);
+        }
+      }
+    }
+  });
+  
+  // corrects the parameter names
+  measurements = correctMeasurementParameter(measurements);
+  // filters out the measurements that are not the latest
+  measurements = getLatestMeasurements(measurements);
+
+  return {
+    name: 'unused',
+    measurements: measurements
+  };
+};
+
+function correctMeasurementParameter(measurements) {
+  
+    measurements.forEach(measurement => {
+      if (measurement.parameter === "pm-10") {
+        measurement.parameter = "pm10";
+      } 
+      else if (measurement.parameter === "pm-25") {
+        measurement.parameter = "pm25";
+      }
+    });
+        return measurements;
+  }
+ 
+  
+function getLatestMeasurements(measurements) {
+    const latestMeasurements = {};
+    
+    measurements.forEach((measurement) => {
+      const key = measurement.parameter + measurement.location;
+      if (!latestMeasurements[key] || measurement.date.local > latestMeasurements[key].date.local) {
+        latestMeasurements[key] = measurement;
+      }
+    });
+  
+    return Object.values(latestMeasurements);
+  }
+
+const stations = [
```

Comment:
Are these still only available statically? are there new ones since the original authoring?

Reply:
The stations are only available statically at this time.